core/templates/dev/head/components/forms/image_uploader_directive.html

```diff
@@ -1,38 +1,101 @@
 <div class="image-uploader-drop-area">
   <div class="image-uploader-text" translate="I18N_DIRECTIVES_DRAG_IMAGE_HERE"></div>
-</div>
-
-<div class="image-uploader-file-input-container">
-  <span translate="I18N_DIRECTIVES_OR_SELECT_FILE"></span>
-  <input type="file" class="image-uploader-file-input" ng-class="fileInputClassName">
+  <br>
+  <span ng-if="errorMessage" class="image-uploader-error-message">
+    <i class="material-icons">&#xE002;</i>
+    <[errorMessage]>
+  </span>
+  <label for="image-file-input" translate="I18N_DIRECTIVES_UPLOAD_A_FILE" class="upload-label-button"></label>
+  <input type="file" id='image-file-input' class="image-uploader-file-input" ng-class="fileInputClassName">
 </div>
 
 <style>
   .image-uploader-drop-area {
     background: #eee;
+    color: black;
+    font-size: 16px;
+    text-align: center;
     height: 200px;
     width: 100%;
+    padding-top: 74px;
+    position: relative;
   }
 
   .image-uploader-text {
-    color: #888;
-    font-size: 2em;
-    height: 100%;
+    display: inline;
+    font-size: 16px;
+    line-height: 28px;
     margin: 0 auto;
-    padding-top: 80px;
     text-align: center;
     width: 100%;
   }
 
   .image-uploader-is-active {
-    background: red;
+    background-color: #bbb;
   }
 
-  .image-uploader-file-input-container {
-    margin-top: 20px;
+  .image-uploader-file-input {
+    width: 0.1px;
+    height: 0.1px;
+    opacity: 0;
+    overflow: hidden;
+    position: absolute;
+    z-index: -1;
   }
 
-  .image-uploader-file-input {
+  label.upload-label-button {
```

Comment:
Can we prefix all this by `image-uploader` in order to make sure they remain scoped to this directive? Ditto below.